numpy/core/src/multiarray/arrayobject.c

```diff
@@ -1645,7 +1645,7 @@ array_new(PyTypeObject *subtype, PyObject *args, PyObject *kwds)
                                      PyArray_BufferConverter,
                                      &buffer,
                                      &offset,
-                                     &PyArray_IntpConverter,
+                                     &PyArray_OptionalIntpConverter,
```

Comment:
error: ‘PyArray_OptionalIntpConverter’ undeclared

Reply:
Any idea why, given it comes from the same file as the previous symbol here?

Reply:
The other symbol is part of the Public API so declared elsewhere.